src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApprovedConsentOrderAboutToSubmitHandler.java

```diff
@@ -72,12 +72,13 @@ public GenericAboutToStartOrSubmitCallbackResponse<Map<String, Object>> handle(C
 
     @Deprecated()
     private void generateAndPrepareDocuments(String authToken, CaseDetails caseDetails, CaseDocument latestConsentOrder) {
-        log.info("Generating and preparing documents for latest consent order, case {}", caseDetails.getId());
+        String caseId = caseDetails.getId().toString();
+        log.info("Generating and preparing documents for latest consent order, case {}", caseId);
 
         Map<String, Object> caseData = caseDetails.getData();
         StampType stampType = documentHelper.getStampType(caseData);
         CaseDocument approvedConsentOrderLetter = consentOrderApprovedDocumentService.generateApprovedConsentOrderLetter(caseDetails, authToken);
```

Comment:
does this also require caseId? consentOrderApprovedDocumentService.generateApprovedConsentOrderLetter(caseDetails, authToken);

Reply:
the id is got from case details